### ISSUE TITLE
Support to see original message

### ISSUE THREAD

**Issue by DevBayer** [MLstate/PEPS#43]
I'm interested with a button/API ACCESS with function to view the original message, with the information like Gmail:

> Delivered-To: ..
> Received: ..
> ..
> Subject: ..
> Content-type: multipart/alternative;..
> Content-Type: text/html; charset="utf-8"

Its possible?

**Comment by hbbio** [comment]
Yes it's possible, the raw message is kept in db. We should add this to todo-list.

**Comment by DevBayer** [comment]
I'm looking the file rawdata.0 but seems manipulated (I'm trying to see the original html content of a message, which an important link and no appears the tag <a>)
